.travis.yml

```diff
@@ -15,6 +19,7 @@ before_install:
 
 install:
     - composer update --prefer-dist
+    - composer require symfony/symfony:${SYMFONY_VERSION}
```

Comment:
```
composer require symfony/symfony:${SYMFONY_VERSION} --no-update
composer update --prefer-dist
```

Will do only one dependencies resolving and prevent from weird Composer errors when it can't update already locked root dependency :)